.github/workflows/test.yaml

```diff
@@ -131,7 +131,8 @@ jobs:
             touch app/emulator.log                            # create log file
             chmod 777 app/emulator.log                        # allow writing to log file
             adb logcat | grep EppoSDK >> app/emulator.log &   # pipe all logcat messages into log file as a background process
-            ./gradlew connectedCheck --no-daemon              # run tests
+            ./gradlew connectedCheck --no-daemon \
+              -Pandroid.testInstrumentationRunnerArguments.TEST_DATA_BRANCH=${{ env.TEST_DATA_BRANCH }}           # run tests
```

Comment:
there are few options available to pass options at runtime to the `connectedCheck`. This is the simplest one I could find.